### PR TITLE
Adding missing Regions and Zones to AWS

### PR DESCRIPTION
**Pull request by bathina2**
## Change Overview

AWS recently added new zones, adding them to our static list.

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [x] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Test Plan

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

**Review by onkarbhat — APPROVED** [review]

File: pkg/blockstorage/awsebs/awsebs.go [hunk 1]
```diff
@@ -757,6 +757,30 @@ func staticRegionToZones(region string) ([]string, error) {
 			"us-west-2-lax-1b",
 			"us-west-2-wl1-sfo-wlz-1",
 		}, nil
+	case "ap-east-1":
```
Comment (onkarbhat) [inline pkg/blockstorage/awsebs/awsebs.go:760]: @bathina2  - I recollect using APIs to discover services(or some sort of catalog of items) from AWS using the EC2 API . Have you checked if there is an API to discover zones ?

Comment (onkarbhat) [reply pkg/blockstorage/awsebs/awsebs.go:760]: @bathina2 - https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/ec2-example-regions-availability-zones.html

Comment (bathina2) [reply pkg/blockstorage/awsebs/awsebs.go:760]: @onkarbhat https://github.com/kastenhq/k10/pull/10445 uses the API to get get the list and compares it with our static list. We didn't want to load the list every time, so this test runs nightly and checks to see if they are in sync

Comment (onkarbhat) [reply pkg/blockstorage/awsebs/awsebs.go:760]: ok, I just realized that this is a fallback mechanism . Nevermind.

Comment (onkarbhat) [reply pkg/blockstorage/awsebs/awsebs.go:760]: Makes sense 👍

**Pull request merged**